### ISSUE TITLE
[Bug Report]: AdventureZ Orc and Industrial Revolution Powered Spawner are incompatible

### ISSUE THREAD

**Issue by Brycey92** [TeamAOF/All-of-Fabric-7#226]
### Modpack Version

2.0.2

### Describe your issue.

I have a Powered Spawner with a Soul Vial containing an Orc. Normally, the Powered Spawner spawns mobs along a predefined spawn area. Every one of the Orcs instead spawn wherever the player is, and with invisibility, and all of them have the NBT tag `OrkSize: 0`.

### Have you modified the modpack?

Yes

### User Modifications

I added WorldEdit.

### Did the issue happen in singleplayer or on a server?

Server

### Discord Username

brycey92

**Comment by Brycey92** [comment]
Soul vial `/kubejs hand`:
```
Item.of('industrialreborn:filled_soul_vial', '{Count:1b,EntityStorage:{Entity:"adventurez:orc"},Favorite:0b,id:"industrialreborn:filled_soul_vial",tag:{Count:1b,EntityStorage:{Entity:"adventurez:orc"},id:"industrialreborn:filled_soul_vial"}}')
```

**Comment by Dmunch04** [comment]
I'm the author of Industrial Reborn, and was recently made aware of the issue. Soul Vials now stores the full entity NBT and initializes them correctly when releasing/spawning. When I tested the changes with AdventureZ orcs it was working correctly. I've released a new version with the fix included (v1.3.1)

**Comment by Trent2087** [comment]
Thanks! Since we are talking Powered Spawners, I put in a Feature Request :)
